src/main/resources/templates/db_scripts/schema.json

```diff
@@ -2,81 +2,18 @@
   "scripts": [
     {
       "run": "after",
-      "snippet": "CREATE OR REPLACE VIEW receiving_history_view AS SELECT pieces.id AS id, json_build_object('id', pieces.jsonb->>'id', 'caption', pieces.jsonb->>'caption', 'comment', pieces.jsonb->>'comment','dateOrdered', purchase_order.jsonb->>'dateOrdered','itemId', pieces.jsonb->>'itemId', 'locationId', pieces.jsonb->>'locationId', 'poLineOrderFormat', po_line.jsonb->>'order_format', 'poLineId', pieces.jsonb->>'poLineId', 'poLineNumber', po_line.jsonb->>'po_line_number', 'purchaseOrderId', po_line.jsonb->>'purchase_order_id', 'poLineReceiptStatus', po_line.jsonb->>'receipt_status', 'receivedDate', pieces.jsonb->>'receivedDate', 'title', po_line.jsonb->>'title', 'receivingNote', details.jsonb->>'receiving_note', 'receivingStatus', pieces.jsonb->>'receivingStatus', 'supplement', pieces.jsonb->>'supplement') AS jsonb FROM pieces LEFT OUTER JOIN po_line ON pieces.jsonb->>'poLineId' = po_line.jsonb->>'id' LEFT OUTER JOIN details ON details.jsonb->>'po_line_id' = po_line.jsonb->>'id' LEFT OUTER JOIN purchase_order ON po_line.jsonb->>'purchase_order_id' = purchase_order.jsonb->>'id';",
+      "snippet": "CREATE OR REPLACE VIEW receiving_history_view AS SELECT pieces.id AS id, json_build_object('id', pieces.jsonb->>'id', 'caption', pieces.jsonb->>'caption', 'comment', pieces.jsonb->>'comment','dateOrdered', purchase_order.jsonb->>'dateOrdered','itemId', pieces.jsonb->>'itemId', 'locationId', pieces.jsonb->>'locationId', 'poLineOrderFormat', po_line.jsonb->>'orderFormat', 'poLineId', pieces.jsonb->>'poLineId', 'poLineNumber', po_line.jsonb->>'poLineNumber', 'purchaseOrderId', po_line.jsonb->>'purchaseOrderId', 'poLineReceiptStatus', po_line.jsonb->>'receiptStatus','receivedDate', pieces.jsonb->>'receivedDate', 'title', po_line.jsonb->>'title','receivingNote', po_line.jsonb->'details'->>'receivingNote', 'receivingStatus', pieces.jsonb->>'receivingStatus', 'supplement', pieces.jsonb->>'supplement') AS jsonb FROM pieces LEFT OUTER JOIN po_line ON pieces.jsonb->>'poLineId' = po_line.jsonb->>'id' LEFT OUTER JOIN purchase_order ON po_line.jsonb->>'purchaseOrderId' = purchase_order.jsonb->>'id';",
       "fromModuleVersion": "1.0"
     }
   ],
   "tables": [
-    {
-      "tableName": "adjustment",
```

Comment:
According to [RMB Tenant API](https://github.com/folio-org/raml-module-builder#the-post-tenant-api)
> `mode` - should be used only to indicate `delete`

Probably it makes sense to specify this mode so the tables will be dropped for existing environments on upgrade stage

Reply:
This PR is already large enough, let's hold off on this and address it afterwards.

Reply:
linked to this from [MODORDSTOR-64](https://issues.folio.org/browse/MODORDSTOR-64) so it doesn't get lost